### DISCUSSION TITLE
v1.3.2

### DISCUSSION THREAD

**Discussion by mkormendy** [mkormendy/homebridge-konnected#50] [Announcements]
First off, apologies to anyone who thought this project was dead. Now, back on track with another update that fixes some very overdue changes that help with upgrading to the latest versions of NodeJS (v20 🎉) and various other node dependencies and security fixes.

### Supported/Tested Versions:
**NodeJS:** v19+
**Homebridge:** v1.7.0

### Bugs and Issues:
If you're having issues with the plugin and things are not working as expected, please [report an issue here](https://github.com/mkormendy/homebridge-konnected/issues/new?assignees=mkormendy&labels=bug&template=bug-report.md&title=).

### Changelog:

- Updates README with Discussions Forum link and cleans up links to Konnected.io unmanaged repo (5eb8577)
- Update bug-report.md (6b28857)
- Updates README Wiki Links (b848626)
- [Updates VS Code ESLint settings, console output, and TypeScript compiler options. (c007a1c)
- Updates node_module dependencies and resolves Dependabot alerts. (691af9b)
- Improves code to "next" ES module-based standards. (e6d47e0)
- Cleanup of logging and renaming of symbol for semantics. (e1c42aa)
- Ensure URL spec format for urls. (9230111)
- Corrects requirement for async/await in promise for fetch request. (5de1284)
- Resolves #40 in response to breaking changes with Node 19+ http agent keepalive defaults which caused implications with the node-fetch module used. (30dbac9)

<hr /><em>This discussion was created from the release <a href='https://github.com/mkormendy/homebridge-konnected/releases/tag/v1.3.2'>v1.3.2</a>.</em>

**Comment by rotax1** [comment]
Thank you for the update. Very happy to see the project is not dead. I am hoping that work on using HomeKit devices as virtual zones becomes a reality.

**Comment by col-forbin** [comment]
Are there plans to update this plugin for Homebridge 2.0 compatibility?

**Comment by mkormendy** [comment]
Yes. But Homebridge 2.0 requires some rewriting of the konnected code.

**Comment by rotax1** [comment]
Looks like we have a release date for Homebridge 2.0 for May the 4th (be with you) which will bring matter support that will can make non-homekit devices appear as matter devices. Should be interesting. Is there any more info on Homebridge 2.0 compatability for the Konnected plugin?  TIA

**Comment by mkormendy** [comment]
I've not touched the project in some time. I am however looking at AI to help me out. Just got myself a Mac Studio to handle some new AI work I've been planning.

**Comment by rotax1** [comment]
Glad to hear you have a plan. I am not a software developer, but however I can help if possible, please let me / us know. Do you think this new Matter support would make using other HomeKit devices as sensors / inputs to the alarm possible or easier.

**Comment by mkormendy** [comment]
I honestly don't have any idea how matter integrates with HomeBridge .. but
I think that the dummy zones might fix that by connecting them to matter
devices with automation if I implemented that feature.

// Mike
